internal/output/logger.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
I don't see a point in having these files outside of `cmd/operator-sdk/bundle/internal`. They should not be used by any other package/command. Having them here implies that they should be.

Reply:
IMO it is generic enough to be used for any other command that would require to do the same and could use a standard for these cases. However, as we have not another command to apply it so far I think it is OK.